askama_derive/build.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    // This build script only exists so that OUT_DIR is set.
+    println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-env-changed=OUT_DIR");
```

Comment:
`cargo:rerun-if-env-changed` reruns only if the environment variable changes (ie, switch targets). It doesn't care if something inside that directory changed. Even with that, I'm not sure having anything here is actually necessary, would need some testing.

Reply:
Seems like none of it is necessary, so I'll just remove the two lines here.

Reply:
Oh! I missed the `env` there!

Reply:
Oh you do still want the first one.

> If the build script inherently does not need to re-run under any circumstance, then emitting cargo:rerun-if-changed=build.rs is a simple way to prevent it from being re-run (otherwise, the default if no rerun-if instructions are emitted is to scan the entire package directory for changes). 

https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed

Reply:
Ah nice. Added it back.